circleguard/visualizer.py

```diff
@@ -76,10 +79,11 @@ def __init__(self, replays=[], beatmap_id=None, beatmap_path=None, parent=None,
             self.hitcircle_radius = circle_radius(self.beatmap.circle_size) - WIDTH_CIRCLE_BORDER / 2
             ## loading stuff
             self.is_loading = True
-            self.sliders_total = 0
+            self.sliders_total = len(self.beatmap.hit_objects) - 1
```

Comment:
why the `-1`?

Reply:
also why is it `sliders_total`, should probably be `num_hitobjs` or something similar? And loading text reference `hitobjects` instead of `sliders`? Not sure why the focus is on sliders when all hitobjects are being processed

Reply:
not all hitobjects are processed, only sliders are: https://github.com/circleguard/circleguard/blob/54e18c4434df027edfabaf5ed21f643f711da62a/circleguard/visualizer.py#L541
There is currently no way to only get sliders atm but I can either filter them here or add a `sliders()` function to slider if you want

Reply:
ah okay, didn't realize. That would be nice - they already have a `circles` method. We can go with the current code for now until there's a new slider release though, even though it's wrong, probably nobody will notice.

Reply:
will add a `sliders()` function to slider, but let's go with this for now.